docs/conf.py

```diff
@@ -78,13 +78,14 @@ def __getattr__(cls, name):
     "iris": ("https://scitools-iris.readthedocs.io/en/latest", None),
     "numpy": ("https://numpy.org/doc/stable", None),
     "scipy": ("https://docs.scipy.org/doc/scipy", None),
-    "numba": ("https://numba.pydata.org/numba-doc/latest", None),
+    "numba": ("https://numba.readthedocs.io/en/latest", None),
```

Comment:
Great catch! However, it looks like Numba uses https://numba.readthedocs.io/en/stable, not /latest. Since packages might switch from stable to latest or vice versa over time, why don't we just link to https://numba.readthedocs.io?

Reply:
Sounds good!

Reply:
I'm reading that it by design won't follow redirects, so you have to choose (but I'll try just in case). If that's the case, stable better?